.github/workflows/fetch-icons.yaml

```diff
@@ -0,0 +1,112 @@
+###
+#
+# Create a pull request with up-to-date icon SVGs
+#
+# This workflow does not have secrets access
+# when run from a fork. Artifacts are uploaded
+# and used in a subsequent workflow with more
+# privileges which does not run unsafe commands.
+#
+# https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
```

Comment:
I don't think this comment is up to date and I also think that this workflow should not run on forks because it uses a github token and the fetchSVG command needs API secrets to be set in the env. We could prevent the workflow from running on a fork with something described here https://github.com/orgs/community/discussions/26409#discussioncomment-3251822 and I think the owner would be 'swisspost' in this case. What do you think?

Reply:
It is a good idea, I just updates this.